docs/samples/edr-api-overview/edr-api-overview.md

```diff
@@ -0,0 +1,136 @@
+| edc.datasource.edr.name                                   | Defines the name associated with the EDR data source | false     | edr           |
+| edc.datasource.edr.url                                    | Defines the database address to access the EDR data  | true      |               |
+| edc.datasource.edr.user                                   | Defines EDR database username                        | true      |               |
+| edc.datasource.edr.password                               | Defines EDR database password                        | true      |               |
+| org.eclipse.tractusx.edc.postgresql.migration.edr.enabled | Enables EDR database migration	                      | true      |               |
+
+## Features
+
+### EDR Negotiation | Contract Negotiation and Transfer Process in a single request
+
+- This endpoint will perform the contract negotiation, transfer process and EDR storage respectively.
+
+> Please note that the `data destination` will always be `HttpProxy`, requiring a request against the provider's `data-plane` to fetch the asset data.
+
+| Path                            | Method | Query Params             |
+|---------------------------------|--------|--------------------------|
+| `<MANAGEMENT_URL>/edrs`         | POST   | none                     |
+
+#### Payload
+
+```json
+{
+  "@context": {
+    "@vocab": "https://w3id.org/edc/v0.0.1/ns/",
+    "odrl": "http://www.w3.org/ns/odrl/2/"
+  },
+  "@type": "NegotiationInitiateRequestDto",
+  "connectorAddress": "<PROVIDER_DSP_URL>",
+  "protocol": "dataspace-protocol-http",
+  "connectorId": "<CONNECTOR_ID>",
+  "providerId": "<PROVIDER_ID>",
+  "offer": {
+    "offerId": "<OFFER_ID>",
+    "assetId": "<ASSET_ID>",
+    "policy": {
+      "@type": "odrl:Set",
+      "odrl:permission": {
+        "odrl:target": "<ASSET_ID>",
+        "odrl:action": {
+          "odrl:type": "USE"
+        }
+      },
+      "odrl:target": "<ASSET_ID>"
+    }
+  }
+}
+```
+
+#### Response
+
+```json
+{
+  "@type": "edc:IdResponseDto",
+  "@id": "contract-negotiation-id",
+  "edc:createdAt": 1687405819736,
+  "@context": {
+    "dct": "https://purl.org/dc/terms/",
+    "tx": "https://w3id.org/tractusx/v0.0.1/ns/",
+    "edc": "https://w3id.org/edc/v0.0.1/ns/",
+    "dcat": "https://www.w3.org/ns/dcat/",
+    "odrl": "http://www.w3.org/ns/odrl/2/",
+    "dspace": "https://w3id.org/dspace/v0.8/"
+  }
+}
+```
+
+### EDR Management | Fetch cached EDRs
+
+- This endpoint will retrieve all EDR entries by their `assetId` or `agreementId` references, which are passed as `query parameters`.
+
+| Path                                         | Method | Query Params         |
+|----------------------------------------------|--------|----------------------|
+| `<MANAGEMENT_URL>/edrs`                      | GET    | assetId, agreementId |
+
+#### Response
```

Comment:
Duplicate header is not allowed and will throw a lint error.
```suggestion
#### EDR Entry Response
```

Reply:
We should adapt the above header as well to have some consistency (have a look at my other comment)